jacoco/src/main/kotlin/org/projectnessie/buildtools/jacoco/JacocoHelperPlugin.kt

```diff
@@ -94,10 +94,16 @@ class JacocoHelperPlugin : Plugin<Project> {
         attribute(DocsType.DOCS_TYPE_ATTRIBUTE, objects.named("source-folders"))
       }
       val sourceSets = project.extensions.getByType<JavaPluginExtension>().sourceSets
-      val mainSourceSet = sourceSets.named("main").get()
+      val mainSourceSet = sourceSets.getByName("main")
       mainSourceSet.java.srcDirs.forEach {
         outgoing.artifact(it) { builtBy(tasks.named(mainSourceSet.classesTaskName)) }
       }
+      val quarkusGeneratedSourceSet = sourceSets.findByName("quarkus-generated-sources")
```

Comment:
Where does the name `quarkus-generated-sources` come from?

Reply:
Quarkus plugin :(

Reply:
They don't add it to the "main" source-set - no clue why

Reply:
TIL